package/MDAnalysis/core/groups.py

```diff
@@ -3032,6 +3032,80 @@ def improper(self):
                 "improper only makes sense for a group with exactly 4 atoms")
         return topologyobjects.ImproperDihedral(self.ix, self.universe)
 
+    @property
+    def ureybradley(self):
+        """This :class:`AtomGroup` represented as an
+        :class:`MDAnalysis.core.topologyobjects.UreyBradley` object
+
+        Raises
+        ------
+        ValueError
+            If the :class:`AtomGroup` is not length 2
+
+
+        .. versionadded:: 0.21.0
+        """
+        if len(self) != 2:
+            raise ValueError(
+                "urey bradley only makes sense for a group with exactly 2 atoms")
+        return topologyobjects.UreyBradley(self.ix, self.universe)
+
+    @property
+    def cmap(self):
+        """This :class:`AtomGroup` represented as an
+        :class:`MDAnalysis.core.topologyobjects.CMap` object
+
+        Raises
+        ------
+        ValueError
+            If the :class:`AtomGroup` is not length 5
+
+
+        .. versionadded:: 0.21.0
+        """
+        if len(self) != 5:
+            raise ValueError(
+                "cmap only makes sense for a group with exactly 5 atoms")
+        return topologyobjects.CMap(self.ix, self.universe)
+
+
+    def convert_to(self, format):
+        """
+        Convert :class:`AtomGroup` to structure from another library.
+
+        Example
+        -------
+
+        .. code-block:: python
+
+            import MDAnalysis as mda
+            from MDAnalysis.tests.datafiles import GRO
+            u = mda.Universe(GRO)
+            parmed_structure = u.atoms.convert_to('PARMED')
```

Comment:
Add sentence stating that `parmed_structure` is now a [ParmEd.structure.Structure](https://parmed.github.io/ParmEd/html/structobj/parmed.structure.Structure.html) instance.

Link to the ParmEd doc.

Add an entry for https://parmed.github.io/ParmEd/html/ to the intersphinx links
https://github.com/MDAnalysis/mdanalysis/blob/972a8b1893b5e353b9d796b15d6fcdd022a93f00/package/doc/sphinx/source/conf.py#L329-L336